src/Agda2Hs/Compile/Utils.hs

```diff
@@ -142,8 +145,12 @@ isClassFunction q
           ExistingClassPragma -> True
           _                   -> False
       _                             -> return False
-  where
-    m = qnameModule q
+
+-- Drops the last (record) module for typeclass methods
+dropClassModule :: ModuleName -> C ModuleName
```

Comment:
This is not used anywhere, did you have something in mind for the future?

Reply:
Not really, I think I was using it at some point but then I refactored something and it was no longer needed.

Reply:
Actually, perhaps we could use it to make sure we do not add qualifiers for record modules.

Reply:
I managed to work around these issues without this function, so I deleted it in the latest push.